cli/deployment/config.go

```diff
@@ -531,6 +531,18 @@ func newConfig() *codersdk.DeploymentConfig {
 			Flag:    "disable-path-apps",
 			Default: false,
 		},
+		SessionDuration: &codersdk.DeploymentConfigField[time.Duration]{
```

Comment:
Question: isn't there a relationship between the`session-duration` and `max-token-lifetime`?

Reply:
SessionDuration is for browser sessions, MaxTokenLifetime is for API keys. When users login to coder (and create a browser session) they cannot change the expiry time. When users create an API key with the CLI command they can specify a custom expiry duration, and the max-token-lifetime governs how long they can make the token last for.